core-bundle/src/Resources/config/listener.yml

```diff
@@ -116,6 +116,19 @@ services:
         arguments:
             - '@database_connection'
 
+    contao.listener.data_container.root_page_dependent_module:
+        class: Contao\CoreBundle\EventListener\DataContainer\RootPageDependentModuleListener
+        public: true
+        arguments:
+            - '@database_connection'
+            - '@translator'
+            - '@contao.routing.scope_matcher'
+            - '@request_stack'
+            - '@Contao\CoreBundle\Csrf\ContaoCsrfTokenManager'
+            - '%contao.csrf_token_name%'
+        tags:
+            - { name: contao.hook, hook: loadDataContainer }
```

Comment:
Why not just the `@Hook()` annotation?

Reply:
Will update. 👍

Reply:
Should only have removed the tag 😂 I'll add it with the change to a more generic solution.